Doc/library/dataclasses.rst

```diff
@@ -292,6 +292,11 @@ Module contents
 
     .. versionadded:: 3.10
 
+   - ``doc``: optional docstring for this field.
+     In CPython it is only used for documenting :attr:`~object.__slots__`.
```

Comment:
Why would this be only true for CPython? Isn't it true for any python implementation that is using this file (dataclasses.py?). I think saying CPython is just confusing things, unless you have some reason for it that I'm not seeing.